### PR TITLE
[PyTorch][Fix] Fix for numerically unstable logsigmoid

### PR DESCRIPTION
**Pull request by crawlingcub**
Fix for #12524 

@masahi

### CODE REVIEW

File: tests/python/frontend/pytorch/test_forward.py [hunk 1]
```diff
@@ -814,6 +814,14 @@ def test_forward_log_sigmoid():
     verify_model(torch.nn.LogSigmoid().eval(), input_data=input_data)
 
 
+@tvm.testing.uses_gpu
+def test_forward_log_sigmoid_overflow():
+    """test_forward_log_sigmoid"""
+    torch.set_grad_enabled(False)
+    input_data =  torch.tensor([-300.0, -100.0).float()
```
Comment (masahi) [inline tests/python/frontend/pytorch/test_forward.py:821]: There is a syntax error.

File: tests/python/frontend/pytorch/test_forward.py [hunk 2]
```diff
@@ -814,6 +814,14 @@ def test_forward_log_sigmoid():
     verify_model(torch.nn.LogSigmoid().eval(), input_data=input_data)
 
 
+@tvm.testing.uses_gpu
+def test_forward_log_sigmoid_overflow():
```
Comment (masahi) [inline tests/python/frontend/pytorch/test_forward.py:818]: Please add a new test under https://github.com/apache/tvm/blob/f64a3bda253f4220d66eeb3348f93f486392cb8e/tests/python/frontend/pytorch/test_forward.py#L814

File: tests/python/frontend/pytorch/test_forward.py (hunk 1 above)
Comment (crawlingcub) [reply tests/python/frontend/pytorch/test_forward.py]: fixed

File: tests/python/frontend/pytorch/test_forward.py (hunk 2 above)
Comment (crawlingcub) [reply tests/python/frontend/pytorch/test_forward.py]: done

**Review by masahi — APPROVED** [review]

**Pull request merged**